distro/run/core/src/main/java/org/camunda/bpm/run/CamundaBpmRunRestConfiguration.java

```diff
@@ -28,15 +25,21 @@
 import org.camunda.bpm.spring.boot.starter.rest.CamundaBpmRestInitializer;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.jersey.JerseyAutoConfiguration;
+import org.springframework.boot.autoconfigure.web.servlet.JerseyApplicationPath;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.boot.web.servlet.FilterRegistrationBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import javax.servlet.Filter;
+
 @EnableConfigurationProperties(CamundaBpmRunProperties.class)
 @Configuration
+@AutoConfigureBefore({ JerseyAutoConfiguration.class })
```

Comment:
Let's remove the annotation:
1. It's semantically incorrect since it should be configured after Jersey
2. Spring Boot seems to be smart enough to figure out the right order of the config classes